apps/mainsite/middleware.py

```diff
@@ -35,6 +39,12 @@ def process_response(self, request, response):
             response.xframe_options_exempt = True
         return response
 
+class ExceptionLoggingMiddleware(MiddlewareMixin):
+    def process_exception(self, request, exception):
+        if django_settings.DEBUG:
+            return None
+        logger.error("An unhandled exception occurred:")
+        logger.exception(exception)
```

Comment:
Just wondering why this is two separate calls.
Both `error` and `exception` use the `ERROR` log level, so I guess this could be `logger.exception("An unhandled exception occured:", exc_info=exception)`?

No need to change, just curious if there is something I am not aware of :-)